test/e2e/framework/pod/pod_client.go

```diff
@@ -58,6 +59,11 @@ const (
 	forbiddenReason = "SysctlForbidden"
 )
 
+// global flags so we can enable features per-suite instead of per-client.
+var (
+	GlobalTrackOwner bool // see WithOwnerTracking
```

Comment:
Can you add a proper godoc comment?

I don't know if people use https://pkg.go.dev/k8s.io/kubernetes/test/e2e/framework/pod, but they might.